pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -58,20 +65,27 @@
 , tabulate
 , tenacity
 , termcolor
+, typing-extensions
 , unicodecsv
 , werkzeug
 , pytestCheckHook
 , freezegun
 , mkYarnPackage
+, writeScript
+
+# Extra airflow providers to enable
+, enabledProviders ? []
```

Comment:
It is forbidden in pythonPackages to conditionally add things to propgatedBuildInputs like this.

Reply:
Well, I'm not sure where we go with this.  We certainly can't enable all the providers, and Airflow is not that useful to many people without some enabled.  I guess @mweinelt's suggestion to move this out of the package tree and make it standalone would resolve the issue for nixpkgs, in our use-case we'd just have to bring it back in via an overlay using `toPythonModule`.